### PR TITLE
adding basic login and confirmation status 1

### PR DESCRIPTION
**Pull request by Elkont33**
Ignore the identity table for now and after deployment I will add the plugins for social omniauth

### CODE REVIEW

**Comment by Elkont33** [comment]
Check this out and create a pull request for your commit as well. I saw it and there are  some files that I don't understand

File: config/application.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Add configuration values here, as shown below.
```
Comment (amrdruid) [inline config/application.yml]: Add this file to the gitignore

File: db/migrate/20160919193901_create_identities.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class CreateIdentities < ActiveRecord::Migration[5.0]
+  def change
+    create_table :identities do |t|
+      t.references :user
```
Comment (amrdruid) [inline db/migrate/20160919193901_create_identities.rb]: `index: true`

File: db/migrate/20160919193901_create_identities.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      t.string :provider
+      t.string :accesstoken
+      t.string :refreshtoken
+      t.string :uid
+      t.string :name
+      t.string :email
```
Comment (amrdruid) [inline db/migrate/20160919193901_create_identities.rb]: Set null with false with the desired columns

File: config/application.yml (hunk 1 above)
Comment (Elkont33) [inline config/application.yml]: Done please create a pull request for your latest commit

**Pull request merged**